lib/suspenders/app_builder.rb

```diff
@@ -209,8 +209,8 @@ def configure_rspec
       copy_file "spec_helper.rb", "spec/spec_helper.rb"
     end
 
-    def configure_travis
-      template 'travis.yml.erb', '.travis.yml'
+    def configure_ci
+      template 'circle.yml.erb', 'circle.yml'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.